### ISSUE TITLE
Loren Ipsum

### ISSUE THREAD

**Issue by shreyasbapat** [shreyasbapat/Test-No-Response#1]
Test

**Comment by shreyasbapat** [comment]
@Juanlu001 Is this behaviour expected?

**Comment by astrojuanlu** [comment]
Kind of! Why did the bot remove the label? Can we customize the message?

**Comment by shreyasbapat** [comment]
Yes, bot removed the label because I replied to it. And yes, we can customise the message

**Comment by shreyasbapat** [comment]
We can create a label like need-more-info . The bot will close every issue with that label if there is no activity in the issue
